### PR TITLE
evaluate rectangle or square 

### PR DESCRIPTION
**Pull request by chetan-7**
### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [ ] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [ ] All filenames are in all uppercase characters with no spaces or dashes.
- [ ] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by chetan-7** [comment]
Added code in java which find whether given parameter form a rectangle or square.

**Comment by shellhub** [comment]
Cloud you reformat your code ?

File: Maths/rectangleorsquare.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import java.util.*;
+class rectangleorsquare {
```
Comment (shellhub) [inline Maths/rectangleorsquare.java:2]: Class name should be Uppercase

File: Maths/rectangleorsquare.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        int d=sc.nextInt();
+        if(a==b && a==c && b==c && c==d && d==b && a==d){
+            //if(a=b=c=d){
+            System.out.println("SQUARE");
+            return;
+        }
+        if(a==b && c==d || a==d && b==c || a==c && b==d ){
+            System.out.println("RECTANGLE");
+        }
+        else System.out.println("NEITHER RECTANGLE OR SQUARE.");
```
Comment (shellhub) [inline Maths/rectangleorsquare.java:17]: We should write these code in function.

**Pull request closed without merging**